integration/common/openlineage/common/provider/dbt/processor.py

```diff
@@ -629,6 +630,8 @@ def extract_namespace(self, profile: Dict) -> str:
             return f"dremio://{profile['software_host']}:{profile['port']}"
         elif self.adapter_type == Adapter.ATHENA:
             return f"awsathena://athena.{profile['region_name']}.amazonaws.com"
+        elif self.adapter_type == Adapter.GLUE:
+            return f"awsglue://glue.{profile['region']}.amazonaws.com"
```

Comment:
We should use glue naming consistent with OL naming schema: https://openlineage.io/docs/spec/naming/#dataset-naming

Spark integration already follows that model.